firmware/controllers/algo/engine2.cpp

```diff
@@ -132,8 +132,8 @@ void EngineState::periodicFastCallback(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	fuelCutoffCorrection = getFuelCutOffCorrection(nowNt, rpm PASS_ENGINE_PARAMETER_SUFFIX);
 
 	// post-cranking fuel enrichment.
-	// for compatibility reasons, apply only if the factor is greater than zero (0.01 margin used)
-	if (engineConfiguration->postCrankingFactor > 0.01f) {
+	// for compatibility reasons, apply only if the factor is greater than unity (only allow adding fuel)
```

Comment:
"for compatibility reasons" it has been a while shall we already drop this if condition altogether? maybe fatal in case of zero or invalid setting?

Reply:
The default, until this PR, has been for that field set to 0, so everyone upgrading a config might still have 0 there.

Reply:
Ohh, so next time.

Reply:
Yep, maybe later we can remove the check.